controller/service.go

```diff
@@ -12,24 +12,40 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+const (
+	birthdayDatabase        = "birthdayDB"
+	birthdayCollection      = "birthdays"
+	personalNumberParameter = "personalNumber"
+	dateParameter           = "date"
+	nameParameter           = "name"
+)
+
+// Service is a type of every method
 type Service struct {
 	Database           *mongo.Database
 	BirthdayCollection *mongo.Collection
+	pb.UnimplementedBirthdayFunctionsServer
 }
 
-type BirthdayStruct struct {
+// BirthdayModel is a model for coverting an object for type: birthdayObject
+type BirthdayModel struct {
 	Name           string
 	Date           string
 	PersonalNumber string
 	ID             string `bson:"_id" json:"id,omitempty"`
 }
 
+// NewService is used each time i create a method
```

Comment:
```suggestion
// NewService creates a Service and returns it.
```
The description is wrong.  You only call it once in the `server.go` file.

Reply:
done